src/sentry/models/grouplink.py

```diff
@@ -77,5 +77,6 @@ class Meta:
         app_label = "sentry"
         db_table = "sentry_grouplink"
         unique_together = (("group", "linked_type", "linked_id"),)
+        indexes = [models.Index(fields=["project", "linked_id", "linked_type", "group"])]
```

Comment:
@wedamija why is `group` part of the index? I don't see it as part of the query for `GroupLink`

Reply:
@scefali It's not required, but adding group here makes this an `INDEX ONLY` query, since the subquery is basically `(select group_id from sentry_grouplink where project in (<projects>) and linked_type = 'something' and linked_id in ('some_id'))`